main.go

```diff
@@ -28,57 +28,61 @@ var usageStr = `
 
 Authorization Options:
-        --user <user>                User required for connections
-        --pass <password>            Password required for connections
-        --auth <token>               Authorization token required for connections
+        --user <user>                               User required for connections
+        --pass <password>                           Password required for connections
+        --auth <token>                              Authorization token required for connections
 
 TLS Options:
-        --tls                        Enable TLS, do not verify clients (default: false)
-        --tlscert <file>             Server certificate file
-        --tlskey <file>              Private key for server certificate
-        --tlsverify                  Enable TLS, verify client certificates
-        --tlscacert <file>           Client certificate CA for verification
+        --tls                                       Enable TLS, do not verify clients (default: false)
+        --tlscert <file>                            Server certificate file
+        --tlskey <file>                             Private key for server certificate
+        --tlsverify                                 Enable TLS, verify client certificates
+        --tlscacert <file>                          Client certificate CA for verification
 
 Cluster Options:
-        --routes <rurl-1, rurl-2>    Routes to solicit and connect
-        --cluster <cluster-url>      Cluster URL for solicited routes
-        --cluster_name <string>      Cluster Name, if not set one will be dynamically generated
-        --no_advertise <bool>        Do not advertise known cluster information to clients
-        --cluster_advertise <string> Cluster URL to advertise to other servers
-        --connect_retries <number>   For implicit routes, number of connect retries
-
+        --routes <rurl-1, rurl-2>                   Routes to solicit and connect
+        --cluster <cluster-url>                     Cluster URL for solicited routes
+        --cluster_name <string>                     Cluster Name, if not set one will be dynamically generated
+        --no_advertise <bool>                       Do not advertise known cluster information to clients
+        --cluster_advertise <string>                Cluster URL to advertise to other servers
+        --connect_retries <number>                  For implicit routes, number of connect retries
+        --cluster_listen <url>                      Cluster url from which members can solicit routes.
 Common Options:
-    -h, --help                       Show this message
-    -v, --version                    Show version
-        --help_tls                   TLS help
+    -h, --help                                      Show this message
+    -v, --version                                   Show version
+        --help_tls                                  TLS help
+Profiling Options:
```

Comment:
Maybe move this section after Cluster Options but before Common Options? (and use new line between sections)